content/participants/ttjc@KumarHemanth342/index.md

```diff
@@ -36,23 +36,93 @@ It would be good if someone tells me how to do auth for apps.
 not a lot of projects are done. I will keep updating it with real links when done.
 My projects:
```

Comment:
Can you differentiate between hosted link and GitHub link in the following format:
_`description`_:
_`Github link`_:
_`hosted link`_:

for every project